certified-connectors/Conversica/apiDefinition.swagger.json

```diff
@@ -0,0 +1,193 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Conversica API",
+    "version": "0.1",
+    "title": "Conversica API",
```

Comment:
title should not contain a work as API, please change the title.

Reply:
I have updated both the title and description.